src/pages/Wallet/WalletOverview/components/WelcomeBanner/WelcomeBanner.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next';
+
+import { XMark } from '@/assets/icons';
+import { Flex } from '@/component-library';
+import { APP_NAME, WRAPPED_TOKEN } from '@/config/relay-chains';
+
+import {
+  StyledCard,
+  StyledCloseCTA,
+  StyledCTALink,
+  StyledImageWrapper,
+  StyledP,
+  StyledSVG,
+  StyledTextWrapper,
+  StyledTitle
+} from './WelcomeBanner.styles';
+
+type WelcomeBannerProps = {
+  onClose: () => void;
+};
+
+const WelcomeBanner = ({ onClose }: WelcomeBannerProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledCard direction='row' alignItems='flex-start' gap='spacing4'>
+      <StyledImageWrapper>
+        <StyledSVG aria-label='bitcoin defi' />
+      </StyledImageWrapper>
+      <StyledTextWrapper direction='column' gap='spacing6'>
+        <Flex direction='column' gap='spacing4'>
+          <StyledTitle weight='bold' size='xl2'>
+            {t('wallet.welcome_to_dapp', { name: APP_NAME })}
+          </StyledTitle>
+          <StyledP size='s' color='tertiary'>
+            {t('wallet.dapp_is_a_one_stop_shop_for_bitcoin_defi', {
+              name: APP_NAME,
+              wrappedToken: WRAPPED_TOKEN.ticker
+            })}
+          </StyledP>
+        </Flex>
+        <Flex gap='spacing4' wrap>
+          <StyledCTALink external icon to={'#'}>
```

Comment:
Which links should I use here @nud3l ?

Reply:
Also, should we dismiss banner per account  or just dismiss it for ever?

Reply:
Just dismiss it forever